tools/tests/karma.conf.js

```diff
@@ -48,6 +49,14 @@ module.exports = function (config) {
       'tooltip/*js',
       'text-field/*.js',
       'accordion-menu/*.js',
+      'interaction-manager/draggable/*.js',
```

Comment:
Cant be just `interaction-manager/**/*.js` or you want to exclude some directories?